schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-variables.xsl

```diff
@@ -134,7 +133,8 @@
     <entry key="http://publications.europa.eu/resource/authority/file-type/GPKG">geopackage</entry>
     <entry key="http://publications.europa.eu/resource/authority/file-type/RSS">georss</entry>
     <entry key="http://publications.europa.eu/resource/authority/file-type/TIFF">image/tiff</entry>
-    <entry key="http://publications.europa.eu/resource/authority/file-type/TIFF">geotiff</entry>
+    <entry key="http://publications.europa.eu/resource/authority/file-type/GeoTIFF">geotiff</entry>
+    <entry key="http://publications.europa.eu/resource/authority/file-type/GeoTIFF">application/x-worldfile</entry>
```

Comment:
These 2 keys are the same, is this correct?

Reply:
Yes the mapping here is to convert distribution format or "WWW:DOWNLOAD:format" protocol values to the EU vocabulary. So various values may be used. Here geotiff are sometimes using `application/x-worldfile` for the world file.